meta-mender-core/classes/mender-helpers.bbclass

```diff
@@ -82,6 +82,30 @@ get_part_number_hex_from_device() {
     fi
 }
 
+def get_part_nummber_padded_from_device(device):
+    return str(''.join(filter(str.isdigit, device))).zfill(2)
+
+def test_part_uuid(d, varName):
+    import re
+    pattern = re.compile("^[0-9a-f]{8}\-[0-9a-f]{4}\-[0-9a-f]{4}\-[0-9a-f]{4}\-[0-9a-f]{12}$")
+    uuid = d.getVar(varName)
+    if len(uuid) == 0:
+       bb.fatal("%s Must be defined" % varName )
+    elif not pattern.match(uuid):
+       bb.fatal("%s Is not a valid PARTUUID.  It most follow the format ########-####-####-####-############.  Allow lower case hex values" % varName )
+    return
+
+def test_disk_identifier(d, varName):
+    import re
+    pattern = re.compile("^[0-9a-f]{8}$")
+    uuid = d.getVar(varName)
+    if len(uuid) == 0:
+       bb.fatal("%s Must be defined" % varName )
+    elif not pattern.match(uuid):
+       bb.fatal("%s Is not a valid Disk Identifier.  It most follow the format ########  Allow lower case hex values" % varName )
+    return
```

Comment:
Please prefix all three of these functions with `mender_`, since this is in a global class.